docs/ref/contrib/admin/index.txt

```diff
@@ -2802,6 +2802,34 @@ creating your own ``AdminSite`` instance (see below), and changing the
 :attr:`AdminSite.index_template` , :attr:`AdminSite.login_template` or
 :attr:`AdminSite.logout_template` properties.
 
+.. _admin-theming:
+
+Theming support
+===============
+
+The admin uses CSS variables to define colors. This allows changing colors
+without having to override many individual CSS rules. For example, if you
+preferred purple instead of blue you could add a ``admin/base.html`` template
+override to your project:
+
+.. code-block:: html+django
+
+    {% extends 'admin/base.html' %}
+
+    {% block extrahead %}{{ block.super }}
+    <style>
+    :root {
+      --primary: #9774d5;
+      --secondary: #785cab;
+      --link-fg: #7c449b;
+      --link-selected-fg: #8f5bb2;
+    }
```

Comment:
Looks like pygments doesn't like the variables here. 

> django/docs/ref/contrib/admin/index.txt:2815: WARNING: Could not lex literal_block as "html+django". Highlighting skipped.

Opened an issue over there. https://github.com/pygments/pygments/issues/1666